demos/blast/blast_in_example.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <seqan/blast.h>
+
+using namespace seqan;
+
+int main()
+{
+    typedef Align<String<AminoAcid>, ArrayGaps> TAlign;
+    typedef BlastMatch<CharString, CharString, uint32_t, TAlign> TBlastMatch;
+    typedef BlastRecord<CharString, CharString, uint32_t, TAlign> TBlastRecord;
+    typedef BlastIOContext<Blosum62> TContext;
```

Comment:
> Do you have to know in advance the correct scoring scheme?

Yes, it is a required spec for the BlastScoringScheme which is in turn required for Blast IO.

> Is Blosum62 reasonable as default?

Since all BLAST programs other than BLASTN score in protein space and Blosum62 is the default there, I think it should be here, as well.